src/RealTime/Localization/Translations/ko.xml

```diff
@@ -129,99 +129,99 @@
-    <translation id="INFO_CONNECTIONS_TOURISTSTOTAL" value="저번 {unit} 중 관광객 방문 : {0:N0}"/>
-    <translation id="INFO_CRIMERATE_CRIMES" value="저번 {unit} 중 검거된 범죄자 수 :"/>
-    <translation id="INFO_ENTERTAINMENT_PARKVISIT" value="저번 {unit} 중 공원과 플라자에 방문한 사람들 : {0:N0}"/>
-    <translation id="INFO_ENTERTAINMENT_UBVISIT" value="저번 {unit} 중 고유 건물에 방문한 사람들 : {0:N0}"/>
-    <translation id="INFO_GARBAGE_INCINERATORCAPACITY" value="쓰레기 처리 용량 : {0:N0} 유닛 / {unit}"/>
-    <translation id="INFO_GARBAGE_GARBAGEPRODUCTION" value="쓰레기 배출: {0:N0} 유닛 / {unit}"/>
+    <translation id="INCOME_TT_COMMERCIALTOURIST" value="저번 {unit} 동안 관광객으로부터 걷은 세금"/>
+    <translation id="INCOME_TT_INDUSTRIAL" value="저번 {unit} 동안 산업구역에서 걷은 세금"/>
+    <translation id="INCOME_TT_OFFICE" value="저번 {unit} 동안 사무구역에서 걷은 세금"/>
+    <translation id="INCOME_TT_PARKAREAS" value="저번 {unit} 동안의 공원 수익"/>
+    <translation id="INCOME_TT_RESIDENTIALHIGH" value="저번 {unit} 동안 주거 지역에서의 최고 세금납부"/>
+    <translation id="INCOME_TT_RESIDENTIALLOW" value="저번 {unit} 동안의 주거 지역의 최저 세금납부"/>
+    <translation id="INCOME_TT_RESIDENTIALTOTAL" value="저번 {unit} 중 주거 지역에서 걷은 세금"/>
+    <translation id="INCOME_TT_TOTAL" value="저번 {unit} 동안 총 수익"/>
+    <translation id="INCOME_TT_TOTAL_COMMERCIAL" value="저번 {unit} 동안 관광객으로 걷은 총 세금"/>
+    <translation id="INCOME_TT_TOTAL_RESIDENTIAL" value="저번 {unit} 동안 주거 지역에서 걷은 총 세금"/>
+    <translation id="INCOME_TT_TOTAL_WORK" value="저번 {unit} 동안 산업과 사무구역에서 걷은 총 세금"/>
+    <translation id="INCOME_TT_PUBLICTRANSPORT" value="저번 {unit} 동안 걷은 대중교통 수익"/>
+    <translation id="INFO_CONNECTIONS_TOURISTSTOTAL" value="저번 {unit} 동안 관광객 방문 : {0:N0}"/>
+    <translation id="INFO_CRIMERATE_CRIMES" value="저번 {unit} 동안 검거된 범죄자 수 :"/>
+    <translation id="INFO_ENTERTAINMENT_PARKVISIT" value="저번 {unit} 동안 공원과 플라자에 방문한 사람들 : {0:N0}"/>
+    <translation id="INFO_ENTERTAINMENT_UBVISIT" value="저번 {unit} 도안 고유 건물에 방문한 사람들 : {0:N0}"/>
+    <translation id="INFO_GARBAGE_INCINERATORCAPACITY" value="쓰레기 처리 용량 : {unit}당 {0:N0}"/>
+    <translation id="INFO_GARBAGE_GARBAGEPRODUCTION" value="쓰레기 배출: {unit}당 {0:N0}"/>
     <translation id="INFO_POPULATION_BIRTHS" value="{0:N0} 출생 / {unit}"/>
     <translation id="INFO_POPULATION_DEATHS" value="{0:N0} 사망 / {unit}"/>
     <translation id="INFO_PUBLICTRANSPORT_COUNT" value="{0:N0} 당 {unit}"/>
-    <translation id="INFO_RES_FORESTAMOUNT" value="{0} 유닛/{unit}"/>
-    <translation id="INFO_RES_FORESTUSED" value="{0} 유닛/{unit}"/>
-    <translation id="INFO_RES_OILUSED" value="{0} 유닛/{unit}"/>
-    <translation id="INFO_RES_OREUSED" value="{0} 유닛/{unit}"/>
-    <translation id="INFO_WATER_CONSUMPTION" value="상수도 사용량 : {0:N0} m³/{unit}"/>
-    <translation id="INFO_WATER_PRODUCTION" value="상수도 가용량 : {0:N0} m³/{unit}"/>
-    <translation id="INFO_WATER_SEWAGEACCUMULATION" value="하수도 사용량 : {0:N0} m³/{unit}"/>
-    <translation id="INFO_WATER_SEWAGECAPACITY" value="하수도 가용량 : {0:N0} m³/{unit}"/>
+    <translation id="INFO_RES_FORESTAMOUNT" value="{0} 유닛/ {unit}"/>
```

Comment:
Why spaces here? Does it improve readability in some way? The original game has no spaces in these places.